.vscode/launch.json

```diff
@@ -6,7 +6,22 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "name": "Run Extension",
+      "name": "Run with default server",
```

Comment:
How do you ensure this configuration is the default one when we package the extension? I don't know if my question is relevant.

Reply:
These configurations are only for development. The "main" method is configured in package.json/main and points to "./dist/extension.js"